Cargo.toml

```
@@ -0,0 +1,8 @@
+[package]
+name = "windows-app"
+version = "0.2.0"
+edition = "2018"
```

Here's what you'll need for publishing.

```suggestion
[package]
name = "windows-app"
version = "0.1.0"
authors = ["Microsoft"]
edition = "2018"
license = "MIT OR Apache-2.0"
description = "Windows App SDK"
repository = "https://github.com/microsoft/windows-app-rs"
documentation = "https://docs.rs/windows-app"
readme = "README.md"
exclude = [".github", ".windows", "docs"]
```